### ISSUE TITLE
Set the most common words in lower case before starting

### ISSUE THREAD

**Issue by parkouss** [parkouss/speed-type#13]
It would be nice to have an option to set the words lower case before starting the common words run.

Maybe with a variable called speed-type-wordlist-transform, with three possible values:

- 'ask to ask if the words should be lower-cased each time
- 'lowercase to lower case automatically
- nil to not transform the text at all

I'd like 'ask to be the default I think.

This proposal is just an idea I have right now, if someone thinks about something better that's tottally doable!

**Comment by dakra** [comment]
Good idea. I'll implement this so you can assign this to me.

Not sure when I find time but hopefully sometime "soon".
